.github/workflows/ci.yaml

```diff
@@ -22,10 +20,15 @@ jobs:
         - ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with: {}
     - name: Update apt-get database
-      run: apt-get update
+      run: sudo apt-get update
```

Comment:
Oh, I remember something about not requiring `sudo` but having to pass an option to the strategy. Can't remember what exactly, but should search my tweets, since I put it there for safe-keeping 😄 (or maybe it wasn't related; can't quite remember).

Reply:
This is dependent on the underlying image. The sane thing to do is to run as an unprivileged user and then use sudo to manipulate the more adminstrative parts inside the image. But the Debian based docker images seem to be running as root, so they don't need the sudo.

setup-beam seem to follow what most other systems do, so the sudo is needed to get access to libsodium.

Reply:
Yeah, but I was sure there was an option to have the image user be root from the get-go, which would be less sane, but wouldn't require `sudo` either 😄 (not sure, though)

Reply:
I like the `sudo ...` prefix. It makes a clear discrimination between "administrative" changes to the environment and what can be done without privileges. In general, the less privileges a build requires, the more I'd trust it to run on a build-host.